src/components/pages/bc-lnbits.ts

```diff
@@ -74,9 +75,17 @@ export class lnbitsPage extends withTwind()(BitcoinConnectElement) {
       return;
     }
 
+    let lnbitsInstanceUrl = this._lnbitsUrl;
+    if (lnbitsInstanceUrl.endsWith('/')) {
```

Comment:
you should parse the URL as we do it in the extension I think. there is probably no need to manually change the URL somehow.